Source/WebCore/accessibility/AccessibilityObject.h

```diff
@@ -938,10 +945,16 @@ class AccessibilityObject : public AXCoreObject {
 
     virtual CommandType commandType() const;
 
+    bool hasRareData() const { return !!m_rareDataWithBitfields.pointer(); }
+    AXObjectRareData* rareData() const { return m_rareDataWithBitfields.pointer(); }
```

Comment:
If this is only meant to be used when hasRareData is true, could this return a CheckedPtr?

Reply:
I guess I'm not seeing the connection between only supposed to being used when hasRareData, and CheckedPtr. CheckedPtr is intended to eagerly cause a crash if the `T` inside is deallocated while a CheckedPtr is on the stack. Forcing all callers of this function into getting a CheckedPtr seems maybe overly restrictive? But maybe I'm missing something.